charts/kong/crds/custom-resource-definitions.yaml

```diff
@@ -1,71 +1,3 @@
-apiVersion: apiextensions.k8s.io/v1beta1
-kind: CustomResourceDefinition
-metadata:
-  name: kongconsumers.configuration.konghq.com
-spec:
-  group: configuration.konghq.com
-  version: v1
-  scope: Namespaced
-  names:
-    kind: KongConsumer
-    plural: kongconsumers
-    shortNames:
-    - kc
-  additionalPrinterColumns:
-  - name: Username
-    type: string
-    description: Username of a Kong Consumer
-    JSONPath: .username
-  - name: Age
-    type: date
-    description: Age
-    JSONPath: .metadata.creationTimestamp
-  validation:
-    openAPIV3Schema:
-      properties:
-        username:
-          type: string
-        custom_id:
-          type: string
-        credentials:
-          type: array
-          items:
-            type: string
----
-apiVersion: apiextensions.k8s.io/v1beta1
-kind: CustomResourceDefinition
-metadata:
-  name: kongcredentials.configuration.konghq.com
-spec:
-  group: configuration.konghq.com
-  version: v1
-  scope: Namespaced
-  names:
-    kind: KongCredential
-    plural: kongcredentials
```

Comment:
My initial instinct here is to not remove this because it means users can't use this resource even for older versions of the controller.
Let's remove it for now and we can add it back in if such a request comes up. I doubt it will come up.

Reply:
Eh, it's kind of in a weird spot. In modern Helm, using the default crds directory behavior, removing it does nothing for existing releases. It's only actually removed when you follow the UPGRADE.md instructions to destroy it.

Old behavior and/or intentionally managing CRDs with Helm 3 (`installCRDs: true`) does destroy it, which is why I tried to write instructions that stress that and provide several ways to check. The riskiness of that management method is, AFAIK, why Helm 3 switched to the new method. It has its own drawbacks, but it's much safer.

That does remind me that I need to instruct people to apply after though--they won't get the status fields otherwise.